src/dcqc/parsers.py

```diff
@@ -0,0 +1,50 @@
+import csv
```

Comment:
In general, I think that most of the modules you have contributed in this suite of PR's could have docstrings so that people can go to the module and have an understanding on a high level of the classes and functions the module contains.

That said, you have done a wonderful job of naming each of your modules.

Reply:
I added some docstrings to classes in #5. Now that the package structure is settled, I feel better about adding docstrings without having to throw them away.